src/controllers/user.js

```diff
@@ -115,3 +115,21 @@ export const getStudents = async (req, res) => {
     return sendMessageResponse(res, 500, 'Server Not Found')
   }
 }
+
+export const addUserToCohort = async (req, res) => {
+  const { student_Id: studentId } = JSON.parse(req.body.body)
+  const { id } = req.params
+  if (!id) {
+    return sendMessageResponse(res, 500, 'Cohort Not Found')
```

Comment:
So this is an example of a "Bad Request". Basically, you're saying the request needs the id. If one is not present, then the submitter of that request hasn't followed the documentation. 

For errors such as these, we use the status 400 (which means bad request). When we use 500, we are saying something is wrong with the server. In this case, that is not true. The server is behaving as intended, it's just a problem with the request that has come in.

Reply:
This has been resolved.